include/stl2/detail/fwd.hpp

```diff
@@ -27,7 +27,7 @@
 #endif // __clang__
 
 #ifndef STL2_WORKAROUND_GCC_69096
- #if defined(__GNUC__) && __GNUC__ >= 6
+ #if defined(__GNUC__) && __GNUC__ >= 6 && __GNUC__ < 10
```

Comment:
We don't support GCC < 7, let's simplify this:
```suggestion
 #if defined(__GNUC__) && __GNUC__ < 10
```

Reply:
I'll apply these in a followup commit since "allow maintainers to push to this branch" isn't enabled.